pkg/core/util.go

```diff
@@ -25,6 +25,36 @@ func CreateGenesisBlock(cfg config.ProtocolConfiguration) (*block.Block, error)
 		return nil, err
 	}
 
+	txs := []*transaction.Transaction{}
+	for i, s := range cfg.GenesisExtensions.Transactions {
+		signers := make([]transaction.Signer, len(s.Signers)+1)
+		signers[0] = transaction.Signer{
+			Account: nextConsensus,
+			Scopes:  transaction.None, // Enough to pay for the transaction.
```

Comment:
`CalledByEntry` is more appropriate. Suppose I want to transfer initial GAS/NEO in this tx.

Reply:
Fixed. Although I had a thought about `All`, what do you think? It's genesis configuration anyway, so that the node operator can perform anything.

Reply:
OK, if it's genesis, it can do anything.